pylops/transforms/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Transforms
+==========
+
+The subpackage tranforms provide various transforms.
+
+A list of available transforms present in pylops.transforms
+
+    DiscreetCosine                  Discreet Cosine Transform
```

Comment:
Should be **Discrete** instead of Discreet everywhere

Reply:
fixed the spelling error everywhere in [af77480](https://github.com/PyLops/pylops/pull/480/commits/af77480d4d0a33b1b27afcc28e70fcefdbc0d6dd)

Reply:
Discreet seems to still be present in few places in the doctoring

Reply:
fixed in [289d51a](https://github.com/PyLops/pylops/pull/480/commits/289d51ad61c7294451d777af6367ea282ad1e78e)